app/pages/organization/organization-page.spec.js

```diff
@@ -7,6 +7,18 @@ import { shallow } from 'enzyme';
 import sinon from 'sinon';
 import OrganizationPage from './organization-page';
 import { organization } from './organization-container.spec';
+import mockPanoptesResource from '../../../test/mock-panoptes-resource';
+
+const noAnnounceCatUrlsOrg = mockPanoptesResource('organizations', {
+  display_name: 'Test Org',
+  description: 'A brief test description',
+  id: '9876',
+  introduction: 'A brief test introduction',
+  links: {
+    organization_roles: ['1']
+  },
+  urls: null
```

Comment:
Interestingly the project resource defaults to an empty array. Is there any chance this might be an empty array? Maybe if someone adds links then deletes them? You might want to check that.

Reply:
Yes great point, though I'm not sure why the organization urls property defaults to null compared to a project urls property default of empty array, an organization urls property will be an empty array if links are added, but then deleted. It was easy to add a related test and add a check on the urls length before rendering the `ExternalLinks` component.